cypress/features/regression/designSystem/content.feature

```diff
@@ -3,7 +3,7 @@ Feature: Content component
 
   @positive
   Scenario Outline: Change children in Content dialog to <children>
-    When I open default "Design System Content Test" component with "content" json from "designSystem" using "<nameOfObject>" object name
+    When I open default "Content Test" component with "content" json from "designSystem" using "<nameOfObject>" object name
```

Comment:
could we also move all `json` files from `designSystem` into `commonComponents` and replace everywhere?

Reply:
I'd like to move these files in a separate PR to make the review easier.